src/common/intl/messages.js

```diff
@@ -1450,6 +1450,9 @@ const messages = {
   logNoStackTrace: 'No stack trace',
   logNoParams: 'No params',
   logDelete: 'Delete log',
+  logDeleteMultiple: 'Delete selected logs',
```

Comment:
I think you need 2 messages here: "Delete selected logs" as a label in the SortedTable and "Delete logs" (which haven't necessarily been selected) in the confirmation modal.